src/test_l1_hdmi_cec_driver.c

```diff
@@ -1191,15 +1187,15 @@ void test_hdmicec_hal_l1_getLogicalAddress_sourceDevice_positive( void )
 
     result = HdmiCecOpen (&handle);
     //if init is failed no need to proceed further
-    UT_ASSERT_EQUAL ( result, HDMI_CEC_IO_SUCCESS );
+    UT_ASSERT_EQUAL_FATAL ( result, HDMI_CEC_IO_SUCCESS );
 
     result = HdmiCecGetLogicalAddress(handle, &logicalAddress);
-    if (HDMI_CEC_IO_SUCCESS != result) { HDMICEC_ASSERT_AUTO_TERM(); UT_FAIL ("HdmiCecGetLogicalAddress failed"); }
-    UT_ASSERT_TRUE(logicalAddress>0x0E && logicalAddress<0x01);
+    if (HDMI_CEC_IO_SUCCESS != result) { HDMICEC_TERM(); UT_FAIL_FATAL ("HdmiCecGetLogicalAddress failed"); }
+    UT_ASSERT_TRUE_FATAL(logicalAddress>0x0E && logicalAddress<0x01);
```

Comment:
logical address should be between 0 and 0x0e for a valid scenario.. we need to have || rather than && to abort the test.

Reply:
here logic it self is wrong if logical address is between 0 & 0x0e
UT_ASSERT_TRUE(logicalAddress>0 && logicalAddress<0x0E);